DESCRIPTION

```diff
@@ -1,14 +1,14 @@
-Package: RWEDataManagementModel
+Package: T2.DMM
 Type: Package
-Title: Data Management Model for Real World Evidence group
+Title: Data Management Model for Real World Evidence
 Version: 0.1.0
 Authors@R: c(
-  person("Albert", "Cid Royo", email = "A.CidRoyo@umcutrecht.nl", role = c("aut", "cre")),
+  person("Albert", "Cid Royo", email = "A.CidRoyo@umcutrecht.nl", role = c("aut", "cre"), comment = c(ORCID = "0000-0002-1863-3450")),
   person("Yinan", "Mao", email = "Y.Mao@umcutrecht.nl", role = "aut"),
   person("Roel", "Elbers", role = "aut"),
   person("Barbara", "Vreede", email = "B.M.I.Vreede@umcutrecht.nl", role = "aut", comment = c(ORCID = "0000-0002-5023-4601")))
 Description: RWE-DataManagementModel is an R package that provides a data model
-  and functionality for processing RWD.
+  and functionality for processing RWD. T2 stands by the semantic harmonization transformation step.
```

Comment:
Hello! Thanks for the review. I have updated the Read.me file.

I did not manually update the `man/T2.DMM-package.Rd ` does it looks like?

Reply:
It could be; as there are files missing from this PR that you probably have locally; specifically the `R/RWEDataManagementModel-package.R` (delete) and `R/T2.DMM-package.R` (add). These need to be added to this PR.